src/openzaak/components/besluiten/api/viewsets.py

```diff
@@ -139,14 +142,60 @@ class BesluitViewSet(
     notifications_kanaal = KANAAL_BESLUITEN
     audit = AUDIT_BRC
 
+    def perform_create(self, serializer):
+        zaak = serializer.validated_data.get("zaak")
+        if zaak and zaak.einddatum is not None:
+            raise PermissionDenied(
+                "Je mag geen gegevens aanpassen van een gesloten zaak."
+            )
```

Comment:
Why is this needed? Can't we rely on super().perform_create(serializer) to make sure the `ClosedZaakMixin` does this?